hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/transport/server/ratis/XceiverServerRatis.java

```diff
@@ -598,6 +605,9 @@ public boolean isExist(HddsProtos.PipelineID pipelineId) {
       for (RaftGroupId groupId : gids) {
         reports.add(PipelineReport.newBuilder()
             .setPipelineID(PipelineID.valueOf(groupId.getUuid()).getProtobuf())
+            .setLeaderID(leaderIdMap.containsKey(groupId) ?
+                ByteString.copyFromUtf8(leaderIdMap.get(groupId).toString()) :
```

Comment:
This does two lookups from leaderIdMap, lets replace this ith leaderIdMap.get and then use the ByteString later.
Also should we add this to te leaderIdmap ?

Reply:
Problem is again shaded ratis ByteString vs google Bytestring. 
So I change the Map to hold ByteString and use that, lookup are cheap though, IMO we are nit-picking :=)